cmd/ocm-addons/installations/cmd.go

```diff
@@ -41,7 +43,7 @@ func generateCommand(options *options, run func(*cobra.Command, []string) error)
 	return cmd
 }
 
-func run(opts *options) func(cmd *cobra.Command, args []string) error {
+func run(opts *options) func(cmd *cobra.Command, args []string) error { //nolint: cyclop
```

Comment:
Just curious, What was the `cyclop` linting error?

Reply:
Cyclomatic complexity was 12 and `cyclop` wants <= 10. A refactor would clear that, but it's still readable at this point so I'm not as worried about it.